store/store.go

```diff
@@ -170,7 +174,7 @@ func assertsURL() string {
 }
 
 func myappsURL() string {
-	if os.Getenv("SNAPPY_USE_STAGING_MYAPPS") != "" {
+	if os.Getenv("SNAPPY_USE_STAGING_CPI") != "" {
```

Comment:
does the same apply to SAS? Maybe we should move them all under a single variable if so.